policies/verify-instances-templates-reference/constraintTemplate.yaml

```
@@ -0,0 +1,38 @@
+---
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata: 
+  name: crownlabsconstraint
```

```suggestion
  name: crownlabs-instance-template-reference
```

I would suggest the same for the CRD name (in the correct case)